titus-server-master/src/main/java/com/netflix/titus/master/mesos/kubeapiserver/direct/DefaultPodAffinityFactory.java

```diff
@@ -181,6 +186,12 @@ private void addNodeAffinityPreferredSelectorConstraint(String key, String value
             term.addMatchExpressionsItem(requirement);
         }
 
+        private void processFarzoneConstraints() {
```

Comment:
same, generalize to any AZs, farzone or not

Reply:
See my comment above ^^^.